src/if.js

```diff
@@ -40,9 +40,34 @@ export class If {
   * @param newValue The new value
   */
   valueChanged(newValue) {
+    if (this.__queuedChanges) {
+      this.__queuedChanges.push(newValue);
```

Comment:
@EisenbergEffect @martingust 
I am reviewing the `if` custom attribute in preparation for an `else` PR.
Is a queue really what we want here?
Let's say that there is a 5 sec animation on entry/exit (awefully long) and a button that toggles visibility.
If I spam the button like crazy, say I click 8 times. Do we really expect a 40 sec animation that does in-out-in-out-in-out?

I think a good UX would rather be: when an animation finish, compare with the current expected state. If it's the same -> done. If it's different add/remove appropriately and repeat the check when the animation (if any) ends.